portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsSubmissions.testcase

```diff
@@ -398,8 +398,6 @@ definition {
 
 		Form.configureSubmitLabel(submitLabel = " ");
 
-		Button.clickDone();
```

Comment:
@Diogomes, can you please do the same on `Forms.testcase`? There is one occurrence like this one. :wink:

Reply:
@natocesarrego  do I send changes made to Forms.testcase by that same LRQA?

Reply:
Yes, @Diogomes. You can add this asked change to this same commit, no worries. :)